README.md

```diff
@@ -107,4 +107,8 @@ we'll use that language.
 1) Fork this repo
 2) Clone your fork to your machine
 3) Add your name to `src/resources/names.txt`, does not need to be your real name, this is a public repo.
+    * Do this by making the change
```

Comment:
This should've been pulled rather than rebased, we can go through it together in office hours if you like later.

Reply:
I saw this a bit late, but I think I get what you mean. I should've pulled and merged in my own branch before pushing it. Need to practice this.